packages/flutter/lib/src/foundation/observer_list.dart

```diff
@@ -137,4 +147,15 @@ class HashedObserverList<T> extends Iterable<T> {
 
   @override
   bool get isNotEmpty => _map.isNotEmpty;
+
+  /// Creates a List containing the elements of the [HashedObserverList].
+  ///
+  /// Overrides the default implementation of [Iterable] to reduce number of
+  /// allocations.
+  @override
+  List<T> toList({bool growable = true}) {
+    final Iterator<T> iter = _map.keys.iterator;
+    return List<T>.generate(_map.length, (_) => (iter..moveNext()).current,
+        growable: growable);
```

Comment:
```suggestion
    final Iterator<T> iterator = _map.keys.iterator;
    return List<T>.generate(
      _map.length,
      (_) => (iterator..moveNext()).current,
      growable: growable,
    );
```

<br>

Aside from whitespace, we might as well also follow the [**avoid abbreviations**](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abbreviations) style guideline.